helm/content-store/templates/configmap.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-nginx-conf
+data:
+  nginx.conf: |-
+    user  nginx;
```

Comment:
Formatting nits (sorry 🙈) - the indentation of the values looks a bit funky here (through to the rest of the file). It looks like they were once lined up with spaces but have since gone wonky 😁

I'd probably just go with one space between the key and the value throughout and save the faff of trying to line things up (I don't think making all the values line up helps readability and it leads to unnecessary diffs and merge conflicts if it's enforced).